### PR TITLE
Fix/more test datasets

### PR DESCRIPTION
**Pull request by ignacioct**
The three tests from modules that needed a change have been adapted to the new Datasets. They were almost functional, nothing but the dataset loading needed a change.

### CODE REVIEW

File: tests/text/modules/heads/test_language_modelling.py [hunk 1]
```diff
@@ -2,14 +2,13 @@
 
 import pandas as pd
 import pytest
-import yaml
-from biome.text import TrainerConfiguration, VocabularyConfiguration
+from biome.text import TrainerConfiguration, VocabularyConfiguration, Dataset
 from biome.text import Pipeline
```
Comment (dcfidalgo) [inline tests/text/modules/heads/test_language_modelling.py:6]: Maybe do this in one import
```suggestion
from biome.text import Pipeline, TrainerConfiguration, VocabularyConfiguration, Dataset
```

File: tests/text/modules/heads/test_record_pair_classification.py [hunk 2]
```diff
@@ -2,14 +2,12 @@
...
-from biome.text import TrainerConfiguration, VocabularyConfiguration
-from biome.text import Pipeline
-from biome.text.data import DataSource
+from biome.text import TrainerConfiguration, VocabularyConfiguration, Dataset, Pipeline
 
 
 @pytest.fixture
-def training_data_source(tmp_path) -> DataSource:
+def training_dataset(tmp_path) -> Dataset:
+    """Creating a dataframe, storing and returning with an intermediate load"""
```
Comment (dcfidalgo) [inline tests/text/modules/heads/test_record_pair_classification.py:10]: See comment above

File: tests/text/modules/heads/test_relation_classifier.py [hunk 3]
```diff
@@ -2,13 +2,12 @@
...
-from biome.text import TrainerConfiguration, VocabularyConfiguration
-from biome.text import Pipeline
-from biome.text.data import DataSource
+from biome.text import TrainerConfiguration, VocabularyConfiguration, Dataset, Pipeline
 
 
 @pytest.fixture
-def training_data_source(tmp_path) -> DataSource:
+def training_dataset(tmp_path) -> Dataset:
+    """Creating a dataframe, storing and returning with an intermediate load"""
```
Comment (dcfidalgo) [inline tests/text/modules/heads/test_relation_classifier.py:10]: Same comment as above

**Review by dcfidalgo — COMMENTED** [review]
Nice, thanks a lot!! Just a few in-line comments to simplify stuff a bit.

File: tests/text/modules/heads/test_language_modelling.py (hunk 1 above)
Comment (ignacioct) [reply tests/text/modules/heads/test_language_modelling.py:6]: Yes, that is much better, I did it in some other files but not in this one for no reason .

**Review by dcfidalgo — APPROVED** [review]
Thanks!

**Pull request merged**